server/src/main/scala/drt/server/feeds/lhr/live/LHRLiveFeed.scala

```diff
@@ -35,36 +36,36 @@ object LHRLiveFeed {
   def flightAndPaxToArrival(lhrArrival: LHRLiveArrival, lhrPax: Option[LHRFlightPax]): Try[Arrival] = {
 
     val tryArrival = Try {
+      val actPax = lhrPax.map(_.TOTALPASSENGERCOUNT.toInt).filter(_!=0)
       Arrival(
-        lhrArrival.OPERATOR,
-        statusCodesToDesc.getOrElse(lhrArrival.FLIGHTSTATUS, lhrArrival.FLIGHTSTATUS),
-        localTimeDateStringToIsoString(lhrArrival.ESTIMATEDFLIGHTOPERATIONTIME),
-        "",
-        localTimeDateStringToIsoString(lhrArrival.ESTIMATEDFLIGHTCHOXTIME),
-        localTimeDateStringToIsoString(lhrArrival.ACTUALFLIGHTCHOXTIME),
-        "",
-        lhrArrival.STAND,
-        lhrPax.map(_.MAXPASSENGERCOUNT.toInt).getOrElse(0),
-        lhrPax.map(_.TOTALPASSENGERCOUNT.toInt).getOrElse(0),
-        lhrPax.map(_.ACTUALTRANSFERPASSENGERCOUNT.toInt).getOrElse(0),
-        "",
-        "",
-        0,
-        "LHR",
-        lhrArrival.TERMINAL,
-        lhrArrival.FLIGHTNUMBER,
-        lhrArrival.FLIGHTNUMBER,
-        lhrArrival.AIRPORTCODE,
-        localTimeDateStringToIsoString(lhrArrival.SCHEDULEDFLIGHTOPERATIONTIME),
-        SDate(localTimeDateStringToIsoString(lhrArrival.SCHEDULEDFLIGHTOPERATIONTIME)).millisSinceEpoch,
-        0,
-        None
+        Operator = if (lhrArrival.OPERATOR.equals("")) None else Some(lhrArrival.OPERATOR),
+        Status = statusCodesToDesc.getOrElse(lhrArrival.FLIGHTSTATUS, lhrArrival.FLIGHTSTATUS),
+        Estimated = if (!StringUtils.isEmpty(lhrArrival.ESTIMATEDFLIGHTOPERATIONTIME)) Try(SDate(localTimeDateStringToIsoString(lhrArrival.ESTIMATEDFLIGHTOPERATIONTIME)).millisSinceEpoch).toOption.filter(_!= 0) else None,
```

Comment:
Rather rename `localTimeDateStringToIsoString` to `localTimeDateStringToMaybeMillis` and have it return Option[MillisSinceEpoch]